tests/aggregation_test.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import pandas as pd
+
+from rdtools.aggregation import aggregation_insol
+
+
+class AggregationTestCase(unittest.TestCase):
+    '''Unit tests for aggregation module'''
+
+    def setUp(self):
+        ind = pd.DatetimeIndex(freq='12h', start='2015-01-01', end='2015-01-02 23:59')
+
+        self.insol = pd.Series(data=[500, 1000, 500, 1000], index=ind)
+        self.energy = pd.Series(data=[1.0, 4, 1.0, 4], index=ind)
+
+        self.aggregated = aggregation_insol(self.energy, self.insol, frequency='D')
+
+    def tearDown(self):
```

Comment:
No need to include a tearDown method if it is not being used. This method is useful if the variables defined in setUp are changed by the individual tests.